### ISSUE TITLE
Full git URL is being logged, including the token

### ISSUE THREAD

**Issue by mateuszpodraza** [jenkins-x/jx-git-operator#23]
The token that has quite broad permissions is being logged without being masked:
```
using poll duration 20s
polling repository jx-boot in namespace jx with git URL https://username:<visible in logs>@github.com/org/repo
cloning repository jx-boot to /tmp/jx-git-operator-606015346/jx-boot
```
If the poll fails, it's logged more than once.

I'd expect the token to be masked in logs - or is this some kind of a configuration mistake on my part?
I followed the getting started guide on jx3.

**Comment by gazal-k** [comment]
I think https://github.com/jenkins-x/jx-git-operator/commit/9252dd5b67eeb06515f2650e2ccf8db594e1c74c fixes this. Haven't verified it in a cluster yet. @anilkumarpasupuleti please let us know once you've verified a newer version?

**Comment by anilkumarpasupuleti** [comment]
@gazal-k, after upgrading the jx-git-operator to latest version i do not see the token being logged anymore. i can confirm that this issues is solved. @mateuszpodraza, This issue has been addressed recently. check it here [https://github.com/jenkins-x/jx-git-operator/commit/9252dd5b67eeb06515f2650e2ccf8db594e1c74c](url)

**Comment by mateuszpodraza** [comment]
Closing as fixed in [v0.0.154](https://github.com/jenkins-x/jx-git-operator/releases/tag/v0.0.154)
